===================================================================
RCS file: /home/cvspsrv/cvsroot/sound/ecasound/pyecasound/Makefile.am,v
retrieving revision 1.28
retrieving revision 1.29
diff -u -r1.28 -r1.29
--- pyecasound/Makefile.am	2003/04/02 20:13:52	1.28
+++ pyecasound/Makefile.am	2003/05/30 01:50:40	1.29
@@ -11,7 +11,7 @@
 ecasoundc_libs 			= -L$(top_builddir)/libecasoundc/.libs -lecasoundc
 endif
 
-if ECA_AM_HAVE_PYTHON
+if ECA_AM_PYECASOUND_INSTALL
 pyecasound_install_list1 = 	$(srcdir)/eci.py \
 				$(srcdir)/pyeca.py \
 				$(srcdir)/ecacontrol.py
@@ -19,7 +19,7 @@
 				$(DESTDIR)$(ECA_S_PYTHON_DLMODULES)/pyeca.py \
 				$(DESTDIR)$(ECA_S_PYTHON_DLMODULES)/ecacontrol.py
 endif
-if ECA_AM_COMPILE_PYECASOUND
+if ECA_AM_PYECASOUND_CEXT
 pyecasound_install_list2 = 	pyecasound.so
 pyecasound_uninstall_list2 =   	$(DESTDIR)$(ECA_S_PYTHON_DLMODULES)/pyecasound.so
 endif
@@ -39,14 +39,14 @@
 # source files
 # ----------------------------------------------------------------------
 
-if ECA_AM_COMPILE_PYECASOUND
+if ECA_AM_PYECASOUND_CEXT
 noinst_LTLIBRARIES 		= libpyecasound.la
 endif
 
 libpyecasound_la_SOURCES	= pyecasound.c
 libpyecasound_la_LDFLAGS 	= -avoid-version -export-dynamic
 
-if ECA_AM_COMPILE_PYECASOUND
+if ECA_AM_PYECASOUND_CEXT
 all: pyecasound.so
 
 pyecasound.so: pyecasound.lo
@@ -57,11 +57,11 @@
 # hooks
 # ----------------------------------------------------------------------
 
-if ECA_AM_COMPILE_PYECASOUND
+if ECA_AM_PYECASOUND_CEXT
 CLEANFILES=pyecasound.so
 endif
 
-if ECA_AM_HAVE_PYTHON
+if ECA_AM_PYECASOUND_INSTALL
 install-exec-hook: $(pyecasound_install_list1) $(pyecasound_install_list2)
 	$(INSTALL) -d $(DESTDIR)$(ECA_S_PYTHON_DLMODULES)
 	$(INSTALL) $(pyecasound_install_list1) $(pyecasound_install_list2) $(DESTDIR)$(ECA_S_PYTHON_DLMODULES)
@@ -69,7 +69,7 @@
 install-exec-hook:
 endif
 
-if ECA_AM_HAVE_PYTHON
+if ECA_AM_PYECASOUND_INSTALL
 uninstall-local:
 	rm -f $(pyecasound_uninstall_list1) $(pyecasound_uninstall_list2)
 	rmdir --ignore-fail-on-non-empty $(DESTDIR)$(ECA_S_PYTHON_DLMODULES)
